### DISCUSSION TITLE
c7n-mailer SQS trigger

### DISCUSSION THREAD

**Discussion by 43976989** [cloud-custodian/cloud-custodian#8935] [AWS] [answered]
Hi,

Anyone know if it's possible to have SQS configured directly as a trigger for the c7n-mailer Lambda, instead of it proxying through EventBridge? It looks like it might possible, and there was at least [some work](https://github.com/cloud-custodian/cloud-custodian/issues/2579) done on this previously, but nothing recent I can find, and testing is also so far unsuccessful. 

Edit: 2nd issue referencing this - [6599](https://github.com/cloud-custodian/cloud-custodian/issues/6599)

Thanks!

**Comment by kapilt** [comment, marked as answer]
Not yet, there's an open issue for it and it's fairly easy since the underlying provisioning framework supports it re contribution
